airflow_home/dags/scripts/citytest_sf_api.py

```diff
@@ -153,7 +153,57 @@ def send_to_color_api(**context):
 
     color = Color()
     formatted = color.format_appointment(appointment)
-    success = Color.patch_appointment(formatted)
+    try:
```

Comment:
moved raise_for_status out here to get sentry logs around send_to_color_api failures.